zeppelin-web/src/app/editor-configuration/editor.configuration.controller.js

```diff
@@ -0,0 +1,231 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
This text of license is different from the one we use in ASF http://www.apache.org/legal/src-headers.html#headers
Can you please use ASF one here and in other places in this PR?